config/environments/production.rb

```diff
@@ -32,7 +32,8 @@
       user_name:            GlobalSetting.smtp_user_name,
       password:             GlobalSetting.smtp_password,
       authentication:       GlobalSetting.smtp_authentication,
-      enable_starttls_auto: GlobalSetting.smtp_enable_start_tls
+      enable_starttls_auto: GlobalSetting.smtp_enable_start_tls,
+      ssl:                  GlobalSetting.smtp_enable_ssl
```

Comment:
Hi @hyunjong-lee, Is this a new setting in Rails 5 because the docs doesn't seem to indicate that `ssl` is a valid option.

![screenshot from 2017-03-14 09-44-11](https://cloud.githubusercontent.com/assets/4335742/23882353/f6730578-089a-11e7-94af-1453bd6d8812.png)